index.js

```diff
@@ -260,7 +260,7 @@ module.exports = {
     if (this._emberVersionRequiresModulesAPIPolyfill()) {
       const ModulesAPIPolyfill = require('babel-plugin-ember-modules-api-polyfill');
 
-      return [[ModulesAPIPolyfill]];
+      return [[ModulesAPIPolyfill, { blacklist: ['@ember/debug'] }]];
```

Comment:
I don't think all of those are required (e.g. this blacklist doesn't prevent `@ember/debug/data-adapter` or `@ember/debug/container-debug-adapter`).  The only one that is not supported by babel-plugin-debug-macros is `Ember.inspect` (which seems misplaced in `@ember/debug` IMHO).